features/networking/pod.feature

```diff
@@ -309,16 +309,14 @@ Feature: Pod related networking scenarios
     Given a pod becomes ready with labels:
       | name=hello-pod |
     And evaluation of `pod` is stored in the :client_pod clipboard
-    # 'yes' command will send a character "h" continously for 3 seconds to /dev/udp on listener where the node is listening for udp traffic on exposed nodeport. The 3 seconds mechanism will create an Assured
```

Comment:
@anuragthehatter I thought your new version should remove line 312 about 'yes' command, but your testing log still has this comment, it's not a big deal, just want to make sure the testing log you post is correct one.

Reply:
Yeah that was taken care before your comment. Thanks for concern though.